zebra/debug_nl.c

```diff
@@ -1713,6 +1713,14 @@ void nl_dump(void *msg, size_t msglen)
 		nlncm_dump(ncm, nlmsg->nlmsg_len - NLMSG_LENGTH(sizeof(*ncm)));
 		break;
 
+	case RTM_NEWQDISC:
+	case RTM_DELQDISC:
+		ncm = NLMSG_DATA(nlmsg);
+		zlog_debug(" ncm [family=%s (%d)]",
+			   af_type2str(ncm->ncm_family), ncm->ncm_family);
+		nlncm_dump(ncm, nlmsg->nlmsg_len - NLMSG_LENGTH(sizeof(*ncm)));
```

Comment:
I don't think this is the correct dump? You will need to write a new one for each RTM_* type you are adding likely